Library/Client.ts

```diff
@@ -142,7 +142,7 @@ class Client {
         ServerRequestTracking.trackRequest(this, request, response, properties);
     }
 
-    public trackDependencyRequest(requestOptions: any, request: http.ClientRequest, properties?: { [key: string]: string; }) {
+    public trackDependencyRequest(requestOptions: http.RequestOptions, request: http.ClientRequest, properties?: { [key: string]: string; }) {
```

Comment:
To be technically correct, I think this needs to be `string | http.RequestOptions | https.RequestOptions`. And same in `ClientRequests.trackRequest()`.

The currently checked-in typings define separate `RequestOptions` classes [for `http`](https://github.com/Microsoft/ApplicationInsights-node.js) and [for `https`](https://github.com/Microsoft/ApplicationInsights-node.js/blob/develop/typings/globals/node/index.d.ts#L567). While they [don't allow the `http.request()` options parameter to be a string](https://github.com/Microsoft/ApplicationInsights-node.js/blob/develop/typings/globals/node/index.d.ts#L425), I consider that a bug in the typings since that API is documented to allow a string and is widely used that way.